flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/regression/MultipleLinearRegression.scala

```diff
@@ -107,6 +107,11 @@ class MultipleLinearRegression extends Predictor[MultipleLinearRegression] {
     this
   }
 
+  def setOptimizationMethod(optimizationMethod: String): this.type = {
```

Comment:
@tillrohrmann How about creating a trait called `OptimizationMethod` which has a 'learningRate' method and using a derived class as a parameter?